File: elementary_computer_science/Python/repetition.py
DNA = input()
DNA_count = []
count = 1
char = DNA[0]
for i in range(1, len(DNA)):
	if DNA[i] == DNA[i-1]:
		count = count + 1
	else:
		DNA_count.append(count)
		count = 1
DNA_count.append(count)
print(max(DNA_count))

File: elementary_computer_science/Cpp/HTP/repetition.cpp
#include <iostream>
using namespace std;

int main(){
	string st;
	long count = 1, max = 1;
	cin >> st;
	if (st.length() == 0) 
		max = 0;
	for (int i = 1; i < int(st.length()); ++i) {
		if (st[i] == st[i-1])
			count += 1;
		else {
			if (count > max)
				max = count;
			count = 1;
		}
	}
	if (count > max)
		max = count;
	cout << max << '\n';
}